apps/writeinpublic/Caddyfile

```diff
@@ -0,0 +1,12 @@
+writeinpublic-test.pa.org.za writeinpublic-test-site1.pa.org.za writeinpublic-test-site2.pa.org.za {
```

Comment:
This is an alternative proxy to nginx. I used it because I was more familiar and it ran against the test instance. This would probably need to be swapped out with dokku managed Nginx @jbothma